joomla/joomla-cms
# [5.1] Convert mod_post_installation_messages to service provider

### Summary of Changes

Converts the administrator post installation messages module to service provider.

### Testing Instructions

1. Make sure you have administrator module mod_post_installation_messages published and have some not hidden post installation messages - click on Reset messages button in Post Installation Messages component
2. You should see module output in administrator area

![mod_post_installation_messages](https://github.com/joomla/joomla-cms/assets/59166858/519c8cfd-d02c-4045-ba59-80b96b99dca4)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
